src/Symfony/Bridge/Doctrine/Attribute/Entity.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\Attribute;
+
+/**
+ * Indicates that a controller argument should receive an Entity.
+ */
+#[\Attribute(\Attribute::TARGET_PARAMETER)]
+class Entity
```

Comment:
I'm wondering whether `Entity` is the right name here if this argument resolver is usable for any doctrine object manager (as only the ORM uses the naming `entity`)

Reply:
`Entity` (and `Repository`), are not only dedicated to doctrine, we also use these words in DDD when talking about a persisted object. 
I think, `Entity` is a better word than `Document` or `Object`, But I get your concern about "doctrine/orm tint" in the word Entity.

@symfony/mergers WDYT?

Reply:
It'd be great to find a common prefix for all attributes that are used by argument resolvers.
Let me give it a try: what about useing `Map*`? Here: `MapEntity`?